README.md

```diff
@@ -220,7 +220,7 @@ groundtruth = DocumentArray(
     Document(uri=d.uri, matches=[Document(uri=d.uri.replace('left', 'right'))]) for d in left_da)
 ```
 
-Here we create a new DocumentArray with real matches by simply replacing the filename, e.g. `left/00001.jpg` to `right/00001.jpg`. That's all we need: if the predicted match has the identical `uri` as the groundtruth match, then it is correct.
+Here we create a new DocArray with real matches by simply replacing the filename, e.g. `left/00001.jpg` to `right/00001.jpg`. That's all we need: if the predicted match has the identical `uri` as the groundtruth match, then it is correct.
```

Comment:
`DocArray` is the package name
DocumentArray is the class/object name. They can not be used interchangeably

Reply:
Understood and fixed